File: 3158. Find the XOR of Numbers Which Appear Twice.go
func duplicateNumbersXOR(nums []int) int {
	answer := 0
	for i := 0; i < len(nums); i++ {
		count := 0
		for j := i; j < len(nums); j++ {
			if nums[i] == nums[j] {
				count += 1
			}
		}
		if count > 1 {
			answer ^= nums[i]
		}
	}
	return answer
}

File: 3158. Find the XOR of Numbers Which Appear Twice.py
class Solution:
    def duplicateNumbersXOR(self, nums: List[int]) -> int:
        answer = 0
        for i in range(len(nums)):
            count = 0
            for j in range(i, len(nums)):
                if nums[i] == nums[j]: count += 1
            if count > 1:
                answer ^= nums[i]
        return answer